packages/core/phase2/exceptions/HO200106.ts

```diff
@@ -1,9 +1,31 @@
+import ExceptionCode from "bichard7-next-data-latest/dist/types/ExceptionCode"
+import errorPaths from "../../lib/exceptions/errorPaths"
 import type { AnnotatedHearingOutcome } from "../../types/AnnotatedHearingOutcome"
 import type Exception from "../../types/Exception"
 import type { ExceptionGenerator } from "../../types/ExceptionGenerator"
 
-const generator: ExceptionGenerator = (_aho: AnnotatedHearingOutcome, _options): Exception[] => {
-  return []
+const generator: ExceptionGenerator = (_aho: AnnotatedHearingOutcome): Exception[] => {
+  const exceptions: Exception[] = []
+
+  const offences = _aho.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant.Offence
+
+  offences.forEach((offence, offenceIndex) => {
+    const results = offence.Result
+
+    results.forEach((result, resultIndex) => {
+      if (!result.PNCAdjudicationExists) {
+        if (!offence.AddedByTheCourt) {
+          const exception = {
+            code: ExceptionCode.HO200106,
+            path: errorPaths.offence(offenceIndex).result(resultIndex).resultClass
+          }
+          exceptions.push(exception)
+        }
+      }
+    })
+  })
```

Comment:
It seems like some of the conditions are missing here.
- In `generateOperations` function, we check if offence and result are recordable. 
- This exception is generated for Sentence result class.
- This exception Is generated when `fixedPenalty` is `false` (check `handleSentence.ts`)